zeebe/exporters/camunda-exporter/src/test/java/io/camunda/exporter/store/ExporterBatchWriterTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.store;
+
+import static io.camunda.zeebe.protocol.record.ValueType.JOB;
+import static io.camunda.zeebe.protocol.record.ValueType.NULL_VAL;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.clearInvocations;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import io.camunda.exporter.entities.TestExporterEntity;
+import io.camunda.exporter.exceptions.PersistenceException;
+import io.camunda.exporter.handlers.TestExportHandler;
+import io.camunda.protocol.TestRecord;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ExporterBatchWriterTest {
+  private ExporterBatchWriter batchWriter;
+  private TestExportHandler handler;
+
+  @BeforeEach
+  void setUp() {
+    handler = mock(TestExportHandler.class);
```

Comment:
:wrench: If we are only using the mock, then there is no need to define a new class `TestExportHandler`.

I suppose you can just use the base class as follows.
```
 private ExportHandler<TestExporterEntity, TestValue> handler;
 handler = mock(ExportHandler.class);
```

Reply:
agreed, I created the class before I decided to use mockito. will remove it.